### ISSUE TITLE
Tooltips don't work consistently across all of HPI

### ISSUE THREAD

**Issue by cmlewis** [tsgrp/hpi#689]

**Comment by gsteimer** [comment]
@cmlewis - Any areas specifically that don't work?  Just wondering where whoever picks this up should start...

**Comment by cmlewis** [comment]
@gsteimer Yep, I'll make a list as I come across things. In general I don't think we init them at the proper time.

**Comment by joehof** [comment]
Christy is spot on - since they are opt-in, you have to initialize them after every render.

I cleaned up the formattribute.js class by adding an afterRender and adding in:

`this.$("span").tooltip();`

for the vierws that used them.

**Comment by mwojno-tsg** [comment]
The issue, https://github.com/tsgrp/HPI/issues/2064, resolves a portion of the tooltips not working consistently across all of HPI, but only focuses on stage, header, and docviewer icons. Below a gif can be seen of the improvements that the above issue's commit added to help get consistency across HPI for tooltips: 


![ToolTipIssueFix](https://user-images.githubusercontent.com/54864662/74742817-eacfab00-5224-11ea-8052-dc31fccde86d.gif)
